backends/advanced/src/advanced_omi_backend/controllers/system_controller.py

```
@@ -284,159 +284,100 @@ async def get_speaker_service_status():
-                "status": "success"
-            }
-        else:
-            return JSONResponse(
-                status_code=500, content={"error": "Failed to reload memory configuration"}
-            )
-        
+        from advanced_omi_backend.model_registry import load_models_config, _find_config_path
+        cfg_path = _find_config_path()
+        load_models_config(force_reload=True)
+        return {"message": "Configuration reloaded", "config_path": str(cfg_path), "status": "success"}
     except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)